lib/http_requests.py

```diff
@@ -76,21 +76,25 @@ def get_http_data(url):
 
 def get_advisories(branch_name):
     url = f"{os.environ['GITHUB_RAW_CONTENT_URL']}/{branch_name}/releases.yml"
-    yml_data = get_http_data(url)['releases']
 
-    advisory_data = []
-    for version in yml_data:
-        try:
-            advisories = yml_data[version]['assembly']['group']['advisories']
-            brew_event = yml_data[version]['assembly']['basis']['brew_event']
-        except:
-            continue
+    try:
+        yml_data = get_http_data(url)['releases']
+
+        advisory_data = []
+        for version in yml_data:
+            try:
+                advisories = yml_data[version]['assembly']['group']['advisories']
+                brew_event = yml_data[version]['assembly']['basis']['brew_event']
+            except:
+                continue
 
-        if -1 in advisories.values() or 1 in advisories.values():
-            continue
-        advisory_data.append([brew_event, advisories])
+            if -1 in advisories.values() or 1 in advisories.values():
+                continue
+            advisory_data.append([brew_event, advisories])
 
-    return sorted(advisory_data, key=lambda x: x[0], reverse=True)
+        return sorted(advisory_data, key=lambda x: x[0], reverse=True)
+    except:
```

Comment:
Always at least scope it to Exception
```suggestion
    except Exception:
```
... otherwise it catches interrupts that you really don't want it to.